Feat/3572  add parent access plugin (#3723)
tracim/tracim: backend/official_plugins/tracim_backend_parent_access/README.md

# Parent Access Plugin

This internal plugin:

> - adds new members of space into all parent (recursively) of this space

Reviewer: ```suggestion
- recursively adds new members of space to its parents
```
